content/en/nodes.md

```diff
@@ -2,14 +2,13 @@
 title: Nodes
 status: Completed
 category: Concept
-tags: ["infrastructure", "fundamental", ""]
+tags: ["infrastructure", "fundamental", "architecture"]
 ---
 
 A node is a computer that works in concert with other computers, or nodes, to accomplish a common task. 
 Take your laptop, modem, and printer, for example. 
 They are all connected over your wifi network communicating and collaborating, each representing one node. 
-In [cloud computing](/cloud-computing/), a node can be a physical computer, 
-a virtual computer, referred to as a [VM](/virtual-machine/), or even a [container](/container/).
+In [cloud computing](/cloud-computing/), a node can be a physical computer, a virtual computer, referred to as a [VM](/virtual-machine/), or even a [container](/container/).
```

Comment:
For reviewers: no content update made here, just a semantic line break adjustment.